From da0aabfd535db2c2d73e0b933ae4afe25c67e8da Mon Sep 17 00:00:00 2001
From: uti <uti.edun@chainguard.dev>
Date: Tue, 25 Feb 2025 19:00:33 +0000
Subject: [PATCH] Update setuptools and pip versions to 75.8.1 and 25.0.1
 respectively. to fix the following CVEs: CVE-2021-3572, CVE-2021-3572,
 CVE-2023-5752, CVE-2024-6345, CVE-2022-40897

---
 Lib/ensurepip/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Lib/ensurepip/__init__.py b/Lib/ensurepip/__init__.py
index 09551411847..b189b5f808e 100644
--- a/Lib/ensurepip/__init__.py
+++ b/Lib/ensurepip/__init__.py
@@ -12,9 +12,9 @@ import tempfile
 __all__ = ["version", "bootstrap"]
 
 
-_SETUPTOOLS_VERSION = "41.2.0"
+_SETUPTOOLS_VERSION = "75.8.1"
 
-_PIP_VERSION = "19.2.3"
+_PIP_VERSION = "25.0.1"
 
 _PROJECTS = [
     ("setuptools", _SETUPTOOLS_VERSION),
-- 
2.48.1

